bcp/client/src/webpack/components/DataLayer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {GeoJSON} from "react-leaflet";
+
+const DataLayer = ({data, pointToLayer, onEachFeature, filter}) => {
```

Comment:
I'm still not clear on the purpose of this component, are you forcing a re-render by changing the key?